docs/changes/coordinates/11469.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Ensure that proper motions can be calculated when converting a ``SkyCoord``
```

Comment:
Why is this file not referring to this PR (11470)?

Reply:
This PR is built on top of #11469, hence it includes its changelog entry. I hope to get #11469 in quickly so I can rebase...

Reply:
Rebased now that #11469 is in, so converting from draft to regular PR.